gulpfile.js

```diff
@@ -115,6 +115,13 @@ gulp.task('build-dist-js', function() {
     .pipe(gulp.dest('dist'));
 });
 
+gulp.task('build-dist-scss', function() {
+  gulp.src('./example/css/react-calendar.scss')
```

Comment:
Yeah! I think it'd make sense. How about keeping `example.scss` where it currently is and moving

`react-calendar.scss` to `src/css/react-calendar.scss`?